ui/src/app/views/workspace-edit/component.html

```diff
@@ -267,7 +267,7 @@ <h3 class="section-header">{{researchPurposeItems.requestReview.shortDescription
                 Clone Workspace
               </button>
             </ng-container>
-            <div *ngIf="!allowSave" class="tooltip-text">
+            <div *ngIf="!allowSave && !savingWorkspace" class="tooltip-text">
```

Comment:
Sort of nitpicky, but I do feel like the functions semantically should be split out:

```
get isValidWorkspace() {
  ...
}

get allowSave() {
    if (this.savingWorkspace) {
      return false;
    }
    return this.isValidWorkspace();
}

...

<div *ngif="!isValidWorkspace" ...
```

Reply:
Done! @calbach PTAL